src/payment/strategies/klarna/klarna-payment-strategy.ts

```diff
@@ -15,12 +18,13 @@ import PaymentMethodActionCreator from '../../payment-method-action-creator';
 import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
 import PaymentStrategy from '../payment-strategy';
 
-import KlarnaCredit, { KlarnaLoadResponse } from './klarna-credit';
+import KlarnaCredit, { KlarnaAddress, KlarnaLoadResponse, KlarnaUpdateSessionParams } from './klarna-credit';
 import KlarnaScriptLoader from './klarna-script-loader';
 
 export default class KlarnaPaymentStrategy implements PaymentStrategy {
     private _klarnaCredit?: KlarnaCredit;
     private _unsubscribe?: (() => void);
+    private _supportedEUCountries = ['AT', 'DE', 'DK', 'FI', 'GB', 'NL', 'NO', 'SE', 'CH'];
```

Comment:
I understand why we do this, but is there a sibling PR in the BE with a list of countries? or the BE is driven by `!= US`?

Reply:
BigPay is using same list => [INT-1398](https://github.com/bigcommerce/bigpay/pull/1585/files#diff-f3036719369d49b5c9e7d5d3c4b3213cR6)